presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/StatisticsBuilder.java

```diff
@@ -20,5 +20,10 @@
 {
     void addBlock(Type type, Block block);
 
+    default void addValue(Type type, Block block, int position)
+    {
+        throw new UnsupportedOperationException();
```

Comment:
I know that adding a default method is a bit controversial, but only some concrete classes implementing this interface need a new method (due to the limit on flat map key type). It didn't make sense to add this method in every existing class, nor it didn't make much sense to introduce a separate interface for this.